### PR TITLE
Verizon SDK 1.6.0 support

### PR DESCRIPTION
**Pull request by jonalter**

### CODE REVIEW

File: ThirdPartyAdapters/verizonmedia/verizonmedia/build.gradle [hunk 1]
```diff
@@ -9,7 +9,7 @@ apply plugin: 'maven-publish'
  */
 ext {
     // String property to store version name.
-    stringVersion = "1.5.0.0"
+    stringVersion = "1.6.0.0"
```
Comment (joshuajplagonera) [inline ThirdPartyAdapters/verizonmedia/verizonmedia/build.gradle:12]: `versionCode` below should also be updated to `1060000`

Comment (jonalter) [reply ThirdPartyAdapters/verizonmedia/verizonmedia/build.gradle:12]: copy. thank you

Comment (jonalter) [reply ThirdPartyAdapters/verizonmedia/verizonmedia/build.gradle:12]: @joshuajplagonera I didn't get a chance to update `versionCode` before it was merged. Would you like me to submit a new PR for that change?

Comment (joshuajplagonera) [reply ThirdPartyAdapters/verizonmedia/verizonmedia/build.gradle:12]: Hi @jonalter 
No worries, we will make this change on the release. Please take note that the `versionCode` should also be updated when updating the `stringVersion`.
Thank you !

**Pull request merged**